superset/assets/src/explore/controls.jsx

```diff
@@ -449,11 +449,16 @@ export const controls = {
   },
 
   order_bars: {
-    type: 'CheckboxControl',
+    type: 'SelectControl',
     label: t('Sort Bars'),
-    default: false,
-    renderTrigger: true,
+    default: 'unsort',
     description: t('Sort bars by x labels.'),
+    renderTrigger: true,
+    choices: formatSelectOptions([
+        'unsort',
```

Comment:
Why are allowing an unsort option? We should just pick `asc` or `desc` as a default and then allow the user to toggle

Reply:
The cancellation sort will be displayed in the original order of the data.